src/client/client4.js

```diff
@@ -2521,22 +2521,6 @@ export default class Client4 {
         );
     };
 
-    // Role Routes
-
-    getRole = async (roleId) => {
```

Comment:
It is good to have these as every API we have still has a corresponding client4 driver.

Reply:
Can you please add these back as it is good to have these as every API we have still has a corresponding client4 driver.